Apps/Examples/Examples/SiwftUI Examples/SwiftUIExamples.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+import UIKit
+@_spi(Experimental) import MapboxMapsSwiftUI
+
+@available(iOS 14.0, *)
+struct SwiftUIExamples: View {
+    var onClose: () -> Void
```

Comment:
Won't `@Environment(\.presentationMode) var presentationMode` and `presentationMode.wrappedValue.dismiss()` 
or 
`@Environment(\.dismiss) private var dismiss`

work instead?